### PR TITLE
feat: track cancel service initiation

### PR DESCRIPTION
**Pull request by subirjolly**
Closes #

### CODE REVIEW

File: src/billing/components/PayAsYouGo/CancellationPanel.tsx [hunk 1]
```diff
@@ -6,12 +6,37 @@ import {
...
+import {getOrg} from 'src/organizations/selectors'
+import {event} from 'src/cloud/utils/reporting'
+
 const CancellationPanel: FC = () => {
   const [isOverlayVisible, setIsOverlayVisible] = useState(false)
+  const quartzMe = useSelector(getQuartzMe)
+  const org = useSelector(getOrg)
+
+  const handleCancelService = () => {
+    if (isFlagEnabled('trackCancellations')) {
```
Comment (joelfried) [inline src/billing/components/PayAsYouGo/CancellationPanel.tsx:27]: Given that Rudderstack isn't loaded into the app withoug the rudderstackReporting flag I'd think we'd want to check for that one here too.

Comment (subirjolly) [reply src/billing/components/PayAsYouGo/CancellationPanel.tsx:27]: makes sense

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 55 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2620)

**Review by joelfried — APPROVED** [review]
LGTM

**Pull request merged**